docs/build/tools/cross-chain/layerzero.md

```diff
@@ -2,277 +2,599 @@
 
-By the end of this guide, you will have transferred tokens from one blockchain to another in a single transaction call using LayerZero Omnichain OFTV1 Contract.
+By the end of this guide, you will have transferred tokens from Kairos Testnet to Base Sepolia in a single transaction call using LayerZero V2 Omnichain Fungible Token (OFT) Contract.
 
-![](/img/build/tools/crosschain-tokens-visuals.png)
+![](/img/build/tools/lz-v2-banner.png)
 
 ## Prerequisites <a id="prerequisites"></a>
 
-Before delving into the entire project, it's important to note that its finished form can be found in this repository: [crosschain-oftv1-example](https://github.com/ayo-klaytn/crosschain-oftv1-example/tree/main). It uses Hardhat, so a prerequisite knowledge will be helpful for understanding how the repository works. Note that this tutorial will not include information on how to use Hardhat and will instead focus solely on the smart contracts. If you would like to follow along, the prerequisites are as follows:
+Before delving into the entire project, it's important to note that its finished form can be found in this repository: [crosschain-oft-v2-example](https://github.com/ayo-klaytn/crosschain-oft-v2-example). It uses Hardhat, so a prerequisite knowledge will be helpful for understanding how the repository works. Note that this tutorial will not include information on how to use Hardhat and will instead focus solely on the smart contracts. If you would like to follow along, the prerequisites are as follows:
 
-* A fresh Hardhat project and [knowledge of how to use Hardhat](../../get-started/hardhat.md)
-* [OpenZeppelin smart contracts installed](https://github.com/OpenZeppelin/openzeppelin-contracts) as a dependency
-* [LayerZero smart contracts installed](https://github.com/LayerZero-Labs/solidity-examples) as a dependency
+* [Knowledge of how to use Hardhat](../../get-started/hardhat.md)
+* Test Funds from both [Kairos Testnet Faucet](https://faucet.kaia.io/) and [Base Sepolia Faucet](https://faucets.chain.link/base-sepolia)
 
-To install both dependencies, you can run:
+## Getting Started <a id="getting-started"></a>
 
-```bash
-npm install @openzeppelin/contracts @layerzerolabs/solidity-examples
-```
+In this guide, we will use LayerZero V2 Omnichain Fungible Tokens (OFT) to facilitate cross-chain token transfer across EVM chains seamlessly. LayerZero provides the [create-lz-oapp](https://github.com/LayerZero-Labs/devtools) CLI toolkit, which we will use to easily build, configure, and deploy our omnichain applications (OApps) on both Kairos Testnet (source chain) and Base Sepolia (destination chain).
 
-## LayerZero OmniChain Contract <a id="layerzero-omnichain-contract"></a>
+## Setting Up Your LayerZero Project <a id="setting-up-your-layerzero-project"></a>
```

Comment:
Why don't you just link like https://docs.layerzero.network/v2/developers/evm/oft/quickstart?

Reply:
imo:
Firstly while i think this is nice, i have made some customization tailored for this guide. 
Secondly, It is also better to explicitly highlight the steps in one place as this tailored for Kaia.

Reply:
@ayo-klaytn The customized guide looks good. But as you know, the contents to be up to date is more important. If they updated the library or guide, it's very hard to chase all these kind of tutorial.

In this case, I think it's better to leave only the guidance like "Kaia specific optimization or customization" with the layerzero's tutorial links. How do you think?

Reply:
Yeah totally agree with you. But in this case, it is only the **Setting Up Your LayerZero Project** section that is void of Kaia specific customization. 

Would you suggest i add the link that points to the setting up project section on L0 docs ?

Reply:
@ayo-klaytn Do you mean https://docs.layerzero.network/v2/developers/evm/create-lz-oapp/start#setting-up-your-layerzero-project?

I think it's enough to note the chain configuration. I can find all tutorial contents in the Layerzero docs.
And we need to inform them the rebranded chain name because it still has [klaytn](https://docs.layerzero.network/v2/developers/evm/technical-reference/deployed-contracts#klaytn) name.

Reply:
Yes 

Wanted opening a PR in their docs, but not sure docs repo is public. 

We'll try reaching out to them.